ApollosProject/apollos-apps
# fix: Daily Habit Mark rendering Active state

## ✏️ Description
After reviewing Figma, I noticed that the Active State for the `DailyHabitsMark` component were incorrect. The solid circle icon should be reserved for days that are _completed_, while the open circle is used for all days that are not yet done.

This PR updates the logic to insure this is correct and matches what we have in Figma

## 🔬 To Test
**Testing Daily Habits**
1. Open up the app
2. Navigate to Daily Habits
3. Interact with Daily Habits to see state changes on the Active day

**Storybook**
1. Set your `PROJECT_VARIANT` to `storybook`
2. Open up the `DailyHabitsMark` story and scroll down towards the bottom

## 📸 Screenshots
| Feature | Screenshot | Design Considerations |
| --- | --- | --- |
| Reference | ![image](https://user-images.githubusercontent.com/45076058/205969334-b05e316d-ceac-4c8a-b1d5-ea94d0cb8439.png) | This is what's in [Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=4341%3A68056&t=gV2jxcqqgTOFaTlb-4) and should be used as the source of truth. |
| Old, incorrect rendering | ![Screenshot_20221206-103820](https://user-images.githubusercontent.com/45076058/205969751-118010d8-5178-4dfb-a2a8-179e3e28bff1.png) |  |
| Updated, fixed Storybook | ![CleanShot 2022-12-06 at 11 24 13](https://user-images.githubusercontent.com/45076058/205969836-5328aa95-7544-4b76-b5dd-17c54726697a.png) | |


## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
